src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -172,23 +182,20 @@ public InternalEngine(EngineConfig engineConfig, boolean skipInitialTranslogReco
     }
 
     /**
-     * Reads the current stored translog ID (v1) from the IW commit data and generates a new/next translog ID (v2)
-     * from the largest present translog ID. If there is no stored translog ID v1 is <code>null</code>
+     * Reads the current stored translog ID from the IW commit data. If the id is not found, recommits the current
+     * translog id into lucene and returns null.
+     *
      */
-    private Tuple<Long, Long> loadTranslogIds(IndexWriter writer, Translog translog) throws IOException {
+    private Long loadCommittedTranslogId(IndexWriter writer, Translog translog) throws IOException {
```

Comment:
Should probably annotate this as `@Nullable` since it can return null, just to make it explicit